code/__defines/misc.dm

```diff
@@ -145,6 +145,8 @@
 #define WALL_CAN_OPEN 1
 #define WALL_OPENING 2
 
+#define MIN_DAMAGE_TO_HIT 5 //Minimum damage needed to dent walls by hitting them with a weapon. 
```

Comment:
This feels too low. That means that stuff like fire extinguishers or toolboxs can just smash metal walls. It should be at least 15.

Reply:
addressed